mpt/core/src/main/java/org/apache/james/mpt/protocol/FileProtocolSessionBuilder.java

```diff
@@ -177,6 +178,11 @@ else if (next.startsWith(SESSION_TAG)) {
                     }
                     sessionNumber = Integer.parseInt(number);
                 }
+                else if (next.startsWith(TIMER)) {
+                    TimerCommand timerCommand = TimerCommand.from(next.substring(6, 11));
```

Comment:
IMO, whatever have been done before, a regexp will be way more clear that what you did. I have to read the full PR to understand how it could work and what it was supposed to do !
Regexp are the tools for what need to be done here